recipe-server/normandy/recipes/api/views.py

```diff
@@ -137,6 +139,53 @@ class RecipeRevisionViewSet(viewsets.ReadOnlyModelViewSet):
         AdminEnabledOrReadOnly,
     ]
 
+    @detail_route(methods=['POST'])
```

Comment:
This viewset doesn't include `permissions.DjangoModelPermissionsOrAnonReadOnly,` in the permissions, so any user can call this detail route, including anonymous users.